src/main/webapp/js/index.js

```diff
@@ -116,16 +116,19 @@ function createReceiptCardElement(receipt) {
 
   // Fill in template fields with correct information.
   const date = new Date(receipt.timestamp);
-  receiptCardClone.querySelector('#timestamp').innerText = date.toDateString();
+  receiptCardClone.querySelector('#timestamp').innerText =
+      date.toUTCString().substring(0, 16);
```

Comment:
nit: this isn't related to categories right? Since it's a small change I'd just mention it in the PR description that you are fixing the date issue too.